codes/quantum/qudits/modular-qudit-913.yml

```diff
@@ -0,0 +1,47 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: modular-qudit-913
+physical: qubits
+logical: qubits
+
+name: 'Modular qudit [[9,1,3]] code'
+introduced: '\cite{arXiv:quant-ph/9610023v2}'
+
+description: 'Generalizing Shor's nine-qubit code, this QECC corrects errors arising from exactly one quantum particle with spin higher than \(1/2\). This code is related to the symmetry group \(\mathbb{Z}_2^{\otimes(N-1)}\otimes S_N\), where \(N\) is the number of states of each spin.'
+
+protection: 'Protects against any quantum error arising from any one of the nine quantum registers. The code distance is \(3\). When \(N = 2\), this code reduces to Shor's nine-qubit code.'
+
+
+features:
+
+  rate: \(1/9\)
+
+  encoders:
+    - 'Copy the state \(|m00000000\rangle\) to \(|m00m00m00\rangle\) using a series of \(CNOT\) gates.'
+    - 'On the first, fourth, and seventh quantum registers, apply quantum discrete Fourier transforms separately \cite{doi:10.1109/SFCS.1994.365700, doi:10.1103/RevModPhys.68.733}. That is, for each \(|m\rangle\), there exists a unitary \(U\)such that \(U|m\rangle = \frac{1}{\sqrt{N}}\sum_{k=0}^{N-1}\omega_{N}^{km}|k\rangle\).'
+    - 'Use a series of \(CNOT\) gates to copy the first register into the second and the third ones, the fourth into the the fifth and the sixth ones, the seventh into the eighth and the ninth ones.'
+    - 'The encoding procedure is summarized below:'
+    \begin{align}
+    |{m00000000}\rangle &\longmapsto |m00m00m00\rangle\\
+                    &\longmapsto \frac{1}{N^{3/2}}\sum_{k,p,q = 0}^{N-1}\omega_N^{m(k+p+q)}|k00p00q00\rangle\\
+                    &\longmapsto \frac{1}{N^{3/2}}\sum_{k,p,q = 0}^{N-1}\omega_N^{m(k+p+q)}|kkkpppqqq\rangle.
+    \end{align}
+
+
+
+
+notes:
+  - 'The condition \([1 + (N^2 -1)n] \leq N^n\) must be satisfied in order to have enough room in the encoded Hilbert space, $n$ is the number of quantum registers.'
+  - 'The code is perfect if the above condition holds \cite{doi:10.1103/PhysRevLett.77.198}.'
+relations:
+  parents:
```

Comment:
parent is qudit_css; please add detail statement if you think frobenius should be cousin. Remove qudits_into_qudits.